src/Raven.Server/Documents/Sharding/Handlers/Processors/DataArchival/ShardedDataArchivalHandlerProcessorForPost.cs

```diff
@@ -0,0 +1,15 @@
+﻿using JetBrains.Annotations;
+using Raven.Client.Exceptions.Sharding;
+using Raven.Server.Documents.Handlers.Processors.DataArchival;
+using Raven.Server.ServerWide.Context;
+
+namespace Raven.Server.Documents.Sharding.Handlers.Processors.DataArchival
+{
+    internal class ShardedDataArchivalHandlerProcessorForPost : AbstractDataArchivalHandlerProcessorForPost<ShardedDatabaseRequestHandler, TransactionOperationContext>
+    {
+        public ShardedDataArchivalHandlerProcessorForPost([NotNull] ShardedDatabaseRequestHandler requestHandler) : base(requestHandler)
+        {
+            throw new NotSupportedInShardingException("Data archival for a sharded database is currently not supported");
```

Comment:
It should work same as expiration / refresh

Reply:
done